packages/kbn-check-mappings-update-cli/current_mappings.json

```diff
@@ -669,6 +669,34 @@
       }
     }
   },
+  "cases-oracle": {
```

Comment:
The PR is going to be merged into a feature branch. There is no need for a thorough review at the moment.